core/system/rule_manager.go

```diff
@@ -26,7 +26,7 @@ type RuleMap map[MetricType][]*Rule
 
 // const
 var (
-	ruleMap    = make(RuleMap)
+	ruleMap    = make(RuleMap, 8)
```

Comment:
initial size 8 is meaningless. 
refer to https://github.com/golang/go/blob/41d8e61a6b9d8f9db912626eb2bbc535e929fefc/src/runtime/map.go#L289
bucketCnt == 8.